server/main.go

```diff
@@ -91,6 +91,8 @@ type IndexHandler struct {
 }
 
 func (h IndexHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+    rw.Header().Set("Content-Security-Policy", "frame-ancestors")
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors
Maybe?
```suggestion
    rw.Header().Set("Content-Security-Policy", "frame-ancestors 'none'")
```

Reply:
I think this caused a problem last time.  I'll have to re-test it